plugins/inputs/couchbase/couchbase.go

```diff
@@ -71,15 +89,26 @@ func (r *Couchbase) gatherServer(addr string, acc telegraf.Accumulator, pool *co
 		}
 		pool = &p
 	}
+
+	sanitizedAddress, err := sanitizeURI(addr)
+	if err != nil {
+		return err
+	}
+
+	if len(sanitizedAddress) <= 1 {
+		log.Printf("I! WARNING: Cluster address tag \"'%s'\" is too short.", sanitizedAddress)
```

Comment:
I'm not sure it's possible to hit this line, if it is can you add a testcase?

You can use "W!" to prefix the log message to indicate a warning.